src/Microsoft.ComponentDetection.Detectors/vcpkg/VcpkgComponentDetector.cs

```diff
@@ -57,7 +60,44 @@ protected override async Task OnFileFoundAsync(ProcessRequest processRequest, ID
             return;
         }
 
-        await this.ParseSpdxFileAsync(singleFileComponentRecorder, file);
+        await this.ParseSpdxFileAsync(this.GetManifestComponentRecorder(singleFileComponentRecorder), file);
+    }
+
+    protected override async Task<IObservable<ProcessRequest>> OnPrepareDetectionAsync(IObservable<ProcessRequest> processRequests, IDictionary<string, string> detectorArgs, CancellationToken cancellationToken = default)
+    {
+        var filteredProcessRequests = new List<ProcessRequest>();
+
+        await processRequests.ForEachAsync(async pr =>
+        {
+            var fileLocation = pr.ComponentStream.Location;
+            var fileName = Path.GetFileName(fileLocation);
+
+            if (fileName.Equals("manifest-info.json", StringComparison.OrdinalIgnoreCase))
```

Comment:
this file name can share the same const variable you create for the other usage below